### PR TITLE
LIVE-4185: update cardano-hw-app

### PR DESCRIPTION
**Pull request by henrily-ledger**
### 📝 Description

Update cardano package because new nano app is not compatible

### ❓ Context

- **Impacted projects**: `live-common` 
- **Linked resource(s)**: `LIVE-4185` 

### ✅ Checklist

- [ ] **Test coverage** 
- [ ] **Atomic delivery** 
- [ ] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: libs/ledger-live-common/package.json [hunk 1]
```diff
@@ -70,7 +70,7 @@
     "updateAppSupportsQuitApp": "node scripts/updateAppSupportsQuitApp.js",
     "prettier": "prettier --write 'src/**/*.?s' 'cli/src/**/*.?s'",
     "lint": "eslint src",
-    "lint:fix": "pnpm lint --fix",
+    "lint:fix": "pnpm lint --fix --quiet",
```
Comment (haammar-ledger) [inline libs/ledger-live-common/package.json:73]: Just curious, is this change required here?

**Review by haammar-ledger — CHANGES_REQUESTED** [review]
@henrily-ledger Can you please:
- add your branch name in the [cardano bot](https://github.com/LedgerHQ/ledger-live/blob/52ecc411d3289f9153c66daba6daaa3ff64d5478/.github/workflows/bot-cardano-silicium.yml#L4), so the bot tests this change
- remove the forcing of the Nano app version in the [bot specs](https://github.com/LedgerHQ/ledger-live/blob/52ecc411d3289f9153c66daba6daaa3ff64d5478/libs/ledger-live-common/src/families/cardano/specs.ts#L30)
- add a changeset

Thanks a lot.

File: libs/ledger-live-common/package.json (hunk 1 above)
Comment (henrily-ledger) [reply libs/ledger-live-common/package.json:73]: My bad, I just sometimes need to only have error, i'll fix

**Review by haammar-ledger — APPROVED** [review]

**Review by hedi-edelbloute — APPROVED** [review]

File: libs/ledger-live-common/src/families/cardano/specs.ts [hunk 2]
```diff
@@ -27,7 +27,6 @@ const cardano: AppSpec<Transaction> = {
   appQuery: {
     model: DeviceModelId.nanoS,
     appName: "CardanoADA",
-    appVersion: "4",
```
[the remark this answers is not in the document]
Comment (gre) [reply libs/ledger-live-common/src/families/cardano/specs.ts:30]: oh it was done

**Comment by gre** [comment]
we were not careful enough : Podfile.lock should not have changed ⚠️  creates issue on PR https://github.com/LedgerHQ/ledger-live/pull/1494 (which i fail to understand why)
